manifests/init.pp

```diff
@@ -266,6 +271,7 @@
   $votequorum_expected_votes           = $::corosync::params::votequorum_expected_votes,
   $quorum_members                      = ['localhost'],
   $quorum_members_ids                  = undef,
+  $quorum_members_names                = undef,
```

Comment:
Even though we haven't switched to data types yet for the whole module, should probably add this as `Optional[Array] $quorum_members_names`